crates/re_viewer/src/ui/view_spatial/scene/scene_part/images.rs

```diff
@@ -146,64 +151,171 @@ impl ImagesPart {
                     return Ok(());
                 }
 
-                let entity_highlight = highlights.entity_highlight(ent_path.hash());
+                if tensor.meaning == TensorDataMeaning::Depth {
+                    if let Some(pinhole_ent_path) = properties.backproject_pinhole_ent_path.as_ref()
+                    {
+                        // NOTE: we don't pass in `world_from_obj` because this corresponds to the
+                        // transform of the projection plane, which is of no use to us here.
+                        // What we want are the extrinsics of the depth camera!
```

Comment:
Hmm I think I understand what's going on, but I don't like it 😄 . Quite unfortunate that we can't use the transform as is